_about/ROADMAP.md

```diff
@@ -0,0 +1,64 @@
+
+In the 2.0.0 release we'll focus on three areas:
+ - Upgrading to [Terraform Plugin SDK v2](https://www.terraform.io/docs/extend/guides/v2-upgrade-guide.html)
+ - [Removing support for Terraform 0.11](https://github.com/hashicorp/terraform-provider-kubernetes/issues/905)
+ - [Changing provider credentialling]
+
+### Coverage
+
+Issues: [theme/coverage](https://github.com/hashicorp/terraform-provider-kubernetes/issues?q=is%3Aopen+is%3Aissue+label%3Atheme%2Fcoverage)
+
+Goal: Improve coverage of the Kubernetes API
+
+While our goal for the Kubernetes provider is complete coverage of the Kubernetes API, this is more work than we can accomplish in one quarter.  Initially, we're prioritizing this work based on user feedback, but we'll also look at ways to address coverage gaps more proactively.
+
+### Lifecycle
+
+Issues: [theme/lifecycle](https://github.com/hashicorp/terraform-provider-kubernetes/issues?q=is%3Aopen+is%3Aissue+label%3Atheme%2Flifecycle)
+
+Goal: Lifecycle improvements to match user workflows
+
+There are several requests to modify behavior based on Kubernetes lifecycle events, for example, waiting or not waiting for a pod to become ready before a deployment is considered complete. We’ll look at this and other areas where additional lifecycle controls make sense in the provider. 
+
+### Errors
+
+Issues: [theme/errors](https://github.com/hashicorp/terraform-provider-kubernetes/issues?q=is%3Aopen+is%3Aissue+label%3Atheme%2Ferrors)
+
+Goal: Improve the error messages generated by the Kubernetes provider and clarify error messages that Kubernetes sends
+
+While there aren't many open issues on this at the moment, one thing we consistently hear an ask for is better error messages.  We will take a holistic look at the error messages in the Kubernetes provider and try to clarify them where possible.
+
+## Research Topics
+
+Research topics include features, architectural changes, and ideas that we are pursuing in the longer term that may significantly impact the core user experience of the Kubernetes provider. Research topics are discovery only and are not guaranteed to be included in a future release.
+
+### Kubernetes Manifest Resource
+
+Issue: [#215](https://github.com/hashicorp/terraform-provider-kubernetes/issues/215)
+
+We’ve been evaluating the best way to support custom resources in the Kubernetes provider. The [initial work](https://github.com/hashicorp/terraform-provider-kubernetes-alpha) led us to create a new provider with a single resource, called `kubernetes_manifest`. We did this because the functionallity needed to accomidate the dynamic schema of custom resources wasn't available in the Terraform SDK.  
+
+Now that the features we need are planned for a near-term SDK release, we're evaluating the feasability of merging the Kubernetes-alpha manifest resource into the existing Kubernetes provider.
```

Comment:
typo - `feasibility`

Reply:
I need a better spell check workflow for my markdown docs, clearly.